### PR TITLE
BZ1873324: seccomp profile changes

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1873324
LGTM from SME in PR #35859 

Releases: 4.6, 4.7, 4.8, 4.9

@sunilcio  - Kindly review the fix.

Preview: https://deploy-preview-36137--osdocs.netlify.app/openshift-enterprise/latest/security/seccomp-profiles.html

### CODE REVIEW

File: modules/applying-custom-seccomp-profile.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[id="applying-custom-seccomp-profile_{context}"]
+= Applying the custom seccomp profile to the workload
+
+.Prerequisite
+* The cluster administrator has set up the custom seccomp profile. For more details, see "Setting up the custom seccomp profile".
+
+.Procedure
+* Apply the seccomp profile to the workload by setting the `securityContext.seccompProfile.type` field as following:
```
Comment (skrthomas) [inline modules/applying-custom-seccomp-profile.adoc:8]: Should it be `seccompProfile` here?

File: modules/applying-custom-seccomp-profile.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++
+[source, yaml]
+----
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: <custom-name>.json <1>
+----
+<1> Provide the name of your custom seccomp profile.
```
Comment (skrthomas) [inline modules/applying-custom-seccomp-profile.adoc:20]: `seccompProfile` here as well?

**Review by skrthomas — COMMENTED** [review]
just a couple of questions about needing to put `seccompProfile` in backticks. I think if you're referring to seccompProfile in a code example, you should use the backticks. If you just need to say seccomp, then no backticks.

File: modules/applying-custom-seccomp-profile.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/applying-custom-seccomp-profile.adoc:8]: Here it just refers to the seccomp as a profile, not the field in the code.

File: modules/applying-custom-seccomp-profile.adoc (hunk 2 above)
Comment (snarayan-redhat) [reply modules/applying-custom-seccomp-profile.adoc:20]: Here too it refers to the profile and not the field in the code. Thanks for the catch though.

**Comment by bmcelvee** [comment]
One additional edit, then we should be ready to merge. Thank you!

File: security/seccomp-profiles.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
+[id="seccomp-profiles"]
+= Configuring seccomp profiles
+include::modules/common-attributes.adoc[]
+:context: configuring-seccomp-profiles
+
+toc::[]
+
+An {product-title} container or a pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
+Seccomp (secure computing mode) is a  Linux kernel feature that can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating a profile that is applied to a container or pod.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply security/seccomp-profiles.adoc]: @bmcelvee - Done

**Pull request merged**